extensions/playerlist/standalone.mjs

```diff
@@ -0,0 +1,46 @@
+// Copyright 2021 HITCON Online Contributors
+// SPDX-License-Identifier: BSD-2-Clause
+
+import assert from 'assert';
+
+/**
+ * This represents the standalone extension service for this extension.
+ */
+class Standalone {
+  /**
+   * Create the standalone extension service object but does not start it.
+   * @constructor
+   * @param {ExtensionHelper} helper - An extension helper object for servicing
+   * various functionalities of the extension.
+   */
+  constructor(helper) {
+    this.helper = helper;
+  }
+
+  /**
+   * Initializes the extension.
+   */
+  async initialize() {
+  }
+
+  /**
+   * Send player list from gameState to client side.
+   * @c2s_getPlayerList
+   * @param {Player} player - player information
+   */
+  async c2s_getPlayerList(player) {
+    const playerIDs = [...this.helper.gameState.players.keys()];
+    await this.helper.callS2cAPI(player.playerID, 'playerlist', 'showPlayerList', 1000, playerIDs);
```

Comment:
Have you considered simply returning the playerIDs? c2s API can return things to the client side.